### PR TITLE
Feature/XIVY-13085 add business notification demo

### PR DESCRIPTION
**Pull request by weissreto**

### CODE REVIEW

File: workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  static {
+    CUSTOMER.setName("Doe");
+    CUSTOMER.setSurname("John");
+    ORDER.setProduct("Ferrari Testarossa");
+  }
+  
+  @Test
+  void runProcess(BpmClient bpmClient)
+  {
```
Comment (ivy-rew) [inline workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java:42]: legacy fromatting ;-)

File: workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json [hunk 2]
```diff
@@ -0,0 +1,385 @@
+{
+  "$schema" : "https://json-schema.axonivy.com/process/11.2.2/process.json",
+  "id" : "18DD058BCE48D418",
+  "config" : {
+    "data" : "workflow.order.OrderData"
+  },
+  "elements" : [ {
```
Comment (ivy-rew) [inline workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json:7]: I'd add an 'annotation' on this process, explaining the purpose of this demo. ... as you see e.g. in procurementRequest

File: workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java (hunk 1 above)
Comment (weissreto) [reply workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java:42]: Can change in a a different PR. But then all demos.

File: workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+      "type" : "TaskEnd",
+      "visual" : {
+        "at" : { "x" : 1408, "y" : 600 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "DialogCall",
+      "name" : "Input Order",
+      "config" : {
+        "dialog" : "workflow.order.InputOrder:start()",
```
Comment (ivy-rew) [inline workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json:48]: the initial order form is empty ... which is realistic, but bad for demonstrations. 
I'd prefil the order form with 'John Doe' and the Ferrari you are also using in tests.
So I can demonstrate the feature without touching the keyboard...

**Review by ivy-rew — APPROVED** [review]
good stuff; some small ideas for improvement

File: workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java (hunk 1 above)
Comment (weissreto) [reply workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java:43]: Have changed this class to legazy formatting only. Will convert every demo to new formatting in a new PR after my holidays.

File: workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+      "connect" : [
+        { "id" : "f30", "to" : "f1" }
+      ]
+    }, {
+      "id" : "f40",
+      "type" : "ProcessAnnotation",
+      "name" : [
+        "**Business Notification Demo**",
+        "",
+        "This demo shows you how you can send business notifications in your process.",
```
Comment (ivy-rew) [inline workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json:371]: double 'you' --> This demo shows how you can ...

File: workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+        "Starts with the simplest way first, then shows how you can use process data in your messages, use multi-lingual messages, and finally how to use templates."
```
Comment (ivy-rew) [inline workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json:372]: It starts ....

**Pull request merged**